modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -32,11 +37,15 @@ continuing. Information about the Operator is displayed.
 . On the *Create Operator Subscription* page:
 .. Select one of the following:
 *** *All namespaces on the cluster (default)* installs the Operator in the default
-*openshift-operators* namespace to watch and be made available to all namespaces
+`openshift-operators` namespace to watch and be made available to all namespaces
 in the cluster. This option is not always available.
 *** *A specific namespace on the cluster* allows you to choose a specific, single
 namespace in which to install the Operator. The Operator will only watch and be
 made available for use in this single namespace.
+ifdef::openshift-dedicated[]
+If you are installing the Cluster Logging Operator, choose this option to select
+the `openshift-logging` namespace.
```

Comment:
@sheriff-rh 	I've added this OSD-only line here so you could presumably use (or link to) this procedure as-is for your Logging-related OSD JIRA, unless there are additional, more specific changes also needed.

Reply:
The cluster logging operator requires openshift-logging namespace for any customer installation.  Guess the difference in OSD is we have created this namespace for the customer.  I'm OK leaving this as-is but wanted to call it out.